cmake/FindLibNeurosim.cmake

```diff
@@ -42,9 +42,9 @@ find_library( PYNEUROSIM_LIBRARY
     )
 
 if ( EXISTS "${LIBNEUROSIM_INCLUDE_DIRS}/neurosim/version.h" )
-  file( STRINGS "${LIBNEUROSIM_INCLUDE_DIRS}/neurosim/version.h" 
+  file( STRINGS "${LIBNEUROSIM_INCLUDE_DIRS}/neurosim/version.h"
                 version_h_contents REGEX "define LIBNEUROSIM_VERSION" )
-  string( REGEX REPLACE ".*([0-9].[0-9].[0-9]).*" "\\1" 
+  string( REGEX REPLACE ".*([0-9]+.[0-9]+.[0-9]+).*" "\\1" 
```

Comment:
@tammoippen 
- Why is the dot between the version digits not escaped here (requiring the dot character), while it is escaped in `FindMusic.cmake`?
- Wouldn't it make sense to define all regular expressions as raw strings, so one avoids the double backslashes?